src/main/java/com/appraisal/modules/employee/controller/EmployeeController.java

```diff
@@ -17,6 +19,7 @@
 @RequiredArgsConstructor
 public class EmployeeController {
     private final EmployeeService employeeService;
+    private final DefaultEmployeeManagerService defaultEmployeeManagerService;
```

Comment:
Always. autowire the interface. Don't couple your classes

Reply:
This is the interface